disttask/loaddata/scheduler.go

```diff
@@ -179,8 +189,12 @@ func init() {
 			if err := json.Unmarshal(bs, &taskMeta); err != nil {
 				return nil, err
 			}
-			logutil.BgLogger().Info("register scheduler constructor", zap.Any("taskMeta", taskMeta))
-			return &ImportScheduler{taskMeta: &taskMeta}, nil
+			logger := logutil.BgLogger().With(zap.String("component", "scheduler"), zap.String("type", proto.LoadData), zap.Int64("ID", taskMeta.JobID))
```

Comment:
use task id? job id will be deleted soon

Reply:
Currently, the task meta does not have a task id, use table id instead now.